ngraph/core/include/openvino/runtime/tensor.hpp

```diff
@@ -149,5 +151,7 @@ class OPENVINO_API Tensor {
      */
     explicit operator bool() const noexcept;
 };
+
+using TensorVector = std::vector<Tensor::Ptr>;
```

Comment:
using TensorVector = std::vector<Tensor>;

Reply:
?! What is the type of vector?

Reply:
Sory:
using TensorVector = std::vector<Tensor>